cv32e40s/tests/programs/custom/b_ext_test/b_ext_test.c

```diff
@@ -1,5 +1,6 @@
 
 #include <stdio.h>
+#include <stdint.h>
```

Comment:
I remember thinking before "why did people say that you need stdint.h, it totally works without it"